ArduPlane/events.cpp

```diff
@@ -55,7 +55,7 @@ void Plane::failsafe_short_on_event(enum failsafe_state fstype, mode_reason_t re
     default:
         break;
     }
-    gcs().send_text(MAV_SEVERITY_INFO, "Flight mode = %u", (unsigned)control_mode);
+    gcs().send_text(MAV_SEVERITY_INFO, "Flight mode = %u", (unsigned)control_mode->mode_number());
```

Comment:
`%s` and `->name4()` would be awesome here :-)

Reply:
sure would! After this PR, and a dozen other related PRs, I'd love to come back and do that!